doc/Core.xml

```diff
@@ -6131,7 +6131,38 @@ onvif://www.onvif.org/name/ARV-453
     </section>
     <section xml:id="_Toc260402083">
       <title>Storage Configuration</title>
-      <para>The following operations allow client to configure the storage configuration data of device. The storage configuration can refer to DAS, NAS, and CDMI Server.</para>
+      <para>
+        The following operations allows a client to configure the storage configuration data of a device.
+      </para>
+      <section>
+        <title>Authentication</title>
+        <para>
+          Depending on the <literal>type</literal> of <literal>StorageConfigurationData</literal>, the device shall interpret fields in a special way:
```

Comment:
Open point: We need to add a paragraph about `ObjectStorageAzure`.

Reply:
We probably will need to. I'll provide feedback on the definition once I'm a bit further in the prototyping.

Reply:
Discussion is ongoing in the other thread, so I will resolve this one.